packages/nextra-theme-docs/src/styles.css

```diff
@@ -383,7 +358,6 @@ article {
       background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' class='h-5 w-5' viewBox='0 0 20 20' fill='currentColor'%3E%3Cpath fill-rule='evenodd' d='M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z' clip-rule='evenodd' /%3E%3C/svg%3E");
       height: 1.2em;
       width: 1.2em;
-      margin-right: 0.125em;
```

Comment:
this margin can be removed for simplify
<img width="81" alt="image" src="https://user-images.githubusercontent.com/7361780/184454220-3b306251-066b-4317-96a2-a332905596ad.png">